scripts/prioritization/assign-r2-priority.js

```diff
@@ -65,27 +65,27 @@ module.exports = async ({ github }) => {
 
  for (const pr of allPRs) {
    try {
-     
+     console.log(`Processing PR #${pr.number}....`);
+
       // Check PR status
-      const isApproved = pr.reviews.nodes.some(
+      const approvals = pr.reviews.nodes.filter(
         (review) => review.state === "APPROVED"
       );
 
-     // Skip if PR is not approved
-      if (!isApproved) {
+      if (approvals.length < 2) {
+        console.log(`Observed approval count: ${approvals.length}. Skipping as it is less than 2...`);
```

Comment:
Should we make it R3 if it only has one approval? Often I think if one community member is reviewing a PR then the others will leave it alone and wait for maintainer